tests/test_xiaomi.py

```diff
@@ -1664,6 +1639,28 @@ async def test_xiaomi_e1_roller_commands_2(zigpy_device_from_quirk, command, val
     )
 
 
+@pytest.mark.parametrize("endpoint", [(1), (2)])
+async def test_aqara_t2_relay(zigpy_device_from_quirk, endpoint):
+    """Test Aqara T2 relay."""
```

Comment:
Note for other reviewers: This test was in a weird place before (between two other related tests), hence it's good to move this.